client/components/tinymce/plugins/media/plugin.jsx

```diff
@@ -13,6 +13,7 @@ import closest from 'component-closest';
 /**
  * Internal dependencies
  */
+import { reduxDispatch } from 'lib/redux-bridge';
```

Comment:
This file is huge so I'm not sure whether it's a problem but we have redux store available already which is defined on [line 47](https://github.com/Automattic/wp-calypso/blob/98e2bca6239f26052ccd43d8bfac9fb72df6ceda/client/components/tinymce/plugins/media/plugin.jsx#L47) from which we can call `store.dispatch`. I only noticed that as I edited that file myself in another PR.

Reply:
Oh gosh! Thanks for pointing that out, I completely missed it.